src/components/BottomNav/BottomNav.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Thunder, Heart, Star } from '../styleElements/icons';
```

Comment:
I keep noticing little things that need to be updated based on the Figma changes